python/orion_finance_sdk_py/contracts.py

```
@@ -1694,46 +1716,117 @@ def set_deposit_access_control(
-    def can_request_deposit(self, user: str) -> bool:
-        """Check if a user is allowed to request a deposit.
-
-        This method queries the vault's depositAccessControl contract.
-        If no access control is set (zero address), it returns True.
-        """
+    def _acl_allows(
+        self,
+        getter,
+        abi_name: str,
+        fn_name: str,
+        *call_args,
```

Ruff (0.16.3)

[warning] Missing type annotation for `*call_args`

(ANN002)